src/main/cms/cms_menu_ledstrip.c

```diff
@@ -59,34 +58,26 @@ const char * const ledProfileNames[LED_PROFILE_COUNT] = {
 
 static long cmsx_Ledstrip_OnEnter(void)
 {
-    if (!featureRead) {
-        cmsx_FeatureLedstrip = featureIsEnabled(FEATURE_LED_STRIP) ? 1 : 0;
-        featureRead = true;
-    }
-
-#ifdef USE_LED_STRIP
+    cmsx_FeatureLedstrip = featureIsEnabled(FEATURE_LED_STRIP) ? 1 : 0;
     cmsx_LedProfile = getLedProfile();
```

Comment:
Won't this yield compilation errors if `USE_LED_STRIP` isn't built?

Reply:
No, the entire thing is wrapped in `#ifdef USE_LED_STRIP` so these are redundant.

Reply:
*mumble mumble* splinter *mumble mumble* beam

Reply:
Falls under the "cleanup" part of the PR. :)

Reply:
Wasn't criticism on your pull request, was criticism on my reviewing perspective. :wink: